pyEPR/project_info.py

```diff
@@ -313,18 +332,34 @@ def connect(self):
         Do establish connection to Ansys desktop.
         Connects to project and then get design and setup
         """
+
         self.connect_project()
-        self.connect_design()
+        if not self.project:
+            logger.info('\tConnection to Ansys NOT established.  \n')
+        if self.project:
+            self.connect_design()
         self.connect_setup()
 
         # Finalize
-        if  self.project:
+        if self.project:
             self.project_name = self.project.name
         if self.design:
             self.design_name = self.design.name
 
-        logger.info(
-            '\tConnection to Ansys established successfully. \N{grinning face} \n')
+        if self.project and self.design:
+            logger.info(
+                '\tConnection to Ansys established successfully. \N{grinning face} \n'
+            )
+
+        if not self.project:
+            logger.info(
+                '\tConnection to project within Ansys not established. Is there a project in your desktop app? \N{thinking face} \n'
+            )
+
+        if not self.design:
+            logger.info(
+                '\tConnection to design within Ansys not established. Is there a design in your project? \N{thinking face} \n'
```

Comment:
This line 361 is incorrect.
1) The connect_ansys() method correctly connects to ansys [in fact I am able to create a design].
2) Point above is a good thing. We should be able to connect to ansys regardless of whether it contains design. In fact we should be able to connect to ansys even if there is no 'project'. And after connecting we create the design or the project.

For the scope of this PR, I only request for this 359-362 to be removed and line 349 to be modified to remove the `and self.design`.

This is the only concern I have on this change. Thus feel free to merge this PR if you accept my comments above (no need to wait for my re-approval). If discussion is needed we can chat tomorrow morning. Cheers